docs/dev/analysis/features/text/pictures-in-runs.rst

```diff
@@ -0,0 +1,105 @@
+Shapes in Runs
+==============
+
+This document is aimed at accessing image information when reading docx files.  Adding image information as part of creating docx files is handled elsewhere.
+
+A Run <w:r> can contain a Drawing <w:drawing>, which can can contain an Inline <wp:inline>, which can contain a Graphic <a:graphic>, which can contain graphicData <a:graphicData>, which can contain a Picture <pic:pic>.  A Picture can contain a blipFill <pic:blipFill> and a "Shape Properties" spPr <pic:spPr>.  A blipFill contains a Blip <a:blip> with an embed property that provides the graphic's Resource ID (rId) needed to look the image up in document.part.rels.  The spPr contains informtion about the image's size.  All of this except the Drawing element are documented in docs/dev/analysis/features/shapes/shapes-inline.rst.
+
+
+Word Behavior
+-------------
+
+In theory, a Run can contain multiple Drawings.  In practice, I have not found a way to create a Run with more than a single Drawing.
+
+
+Protocol
+--------
+
+.. highlight:: python
+
+To stay parallel with document.add_picture() behavior, which creates an Inline object, the Run should have an Inlines property, producing a list of Inline objects.
+
+  >>> inlines = run.inline_lst
+  >>> inline = inlines[0]
+  >>> inline
+  <CT_Inline '<wp:inline>' at 0x43de510>
```

Comment:
The protocol expresses the behavior of the library from the perspective of the user (a developer). The user doesn't deal directly with the oxml layer, that's an internal implementation detail. So minimally, this sort of thing belongs in a different section. I don't typically document the oxml design, I just work it out when I get there, mostly because it's so convention-driven there's usually not a lot to decide on. But if you want this, just segregate it into a section lower down, maybe 'Oxml Implementation' or something.